src/main/java/access/messaging/AccessThreadManager.java

```diff
@@ -65,134 +54,31 @@
-	}
-
-	/**
-	 * Initializes consumer listeners for the thread pool workers.
-	 */
-	@PostConstruct
-	public void initialize() {
-		// Initialize the Kafka Producer
-		producer = KafkaClientFactory.getProducer(kafkaAddress);
-
-		// Initialize the Map of running Threads
-		runningJobs = new HashMap<>();
-
-		// Start polling for Kafka Jobs on the Group Consumer.
-		// Occurs on a separate Thread to not block Spring.
-		new Thread(() -> pollAccessJobs()).start();
-
-		// Start polling for Kafka Abort Jobs on the unique Consumer.
-		new Thread(() -> pollAbortJobs()).start();
-	}
-
-	/**
-	 * Opens up a Kafka Consumer to poll for all Access Jobs that should be processed by this component.
-	 */
-	public void pollAccessJobs() {
-		
-		Consumer<String, String> generalConsumer = null;
-		
-		try {
-			// Callback that will be invoked when a Worker completes. This will
-			// remove the Job Id from the running Jobs list.
-			WorkerCallback callback = jobId -> runningJobs.remove(jobId);
-
-			// Create the General Group Consumer
-			generalConsumer = KafkaClientFactory.getConsumer(kafkaAddress, kafkaGroup);
-			List<String> topics = Arrays.asList(String.format(LOGGER_FORMAT, ACCESS_TOPIC_NAME, space));
-			// Log topics we're listening to
-			pzLogger.log(String.format("Begin listening to Kafka topics : %s", String.join(", ", topics)), Severity.INFORMATIONAL);
-			// Listen
-			generalConsumer.subscribe(topics);
-
-			// Poll
-			while (!closed.get()) {
-				ConsumerRecords<String, String> consumerRecords = generalConsumer.poll(1000);
-				// Handle new Messages on this topic.
-				for (ConsumerRecord<String, String> consumerRecord : consumerRecords) {
-					pzLogger.log(String.format("Processing Job ID %s on Access Worker Thread.", consumerRecord.key()),
-							Severity.INFORMATIONAL);
-					// Create a new worker to process this message and add it to
-					// the thread pool.
-					Future<?> workerFuture = accessWorker.run(consumerRecord, producer, callback);
-
-					// Keep track of all Running Jobs
-					runningJobs.put(consumerRecord.key(), workerFuture);
-				}
-			}
-		} catch (WakeupException exception) {
-			String error = String.format("Polling Thread forcefully closed: %s", exception.getMessage());
-			LOGGER.error(error, exception, new AuditElement("access", "kafkaListenerShutDown", ""));
-			pzLogger.log(error, Severity.ERROR);
-		}
-		finally {
-			if( generalConsumer != null ) {
-				generalConsumer.close();
-			}
-		}
-	}
 
 	/**
-	 * Begins listening for Abort Jobs. If a Job is owned by this component, then it will be terminated.
+	 * Processes an Abort Job message coming in through the queue. If this component contains that job, it will cancel
+	 * it.
+	 * 
+	 * @param abortJobRequest
+	 *            The request containing the information about the Job cancellation request
 	 */
-	public void pollAbortJobs() {
-		
-		Consumer<String, String> uniqueConsumer = null;
-		
+	@RabbitListener(queues = "Abort-Job-${SPACE}")
```

Comment:
I see queues "AccessJob-SPACE" and "Abort-Job-SPACE". I would pick a naming convention and use it throughout (one has an additional hyphen).